stdlib/blocktx.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: Apache-2.0
+
+function blockhash(uint blockNumber) returns (bytes32 ret) {
+  assembly {
+    ret := blockhash(blockNumber)
+  }
+}
+
+function gasleft() returns (uint256 ret) {
+  assembly {
+    ret := gas()
+  }
+}
+
+library Block {
```

Comment:
Instead of libraries, we can split these into different files and import them based on file names.

Reply:
Yes, I think we should split this in many small files.

Reply:
(mainly to test out if the way we plan to do namespacing in the future works out well or not)